src/g_all_guis.h

```diff
@@ -216,7 +215,6 @@ typedef struct _vslider
 {
     t_iemgui x_gui;
     int      x_pos;
-    int      x_val;
```

Comment:
i wonder why you kept `x_pos` rather than `x_val`.
isn't the latter what we actually want (the current *value* of the slider, not its position)

Reply:
Well I think it's more really its position, [0;1] normalized. 
The "value" involves scaling and optional log transformation, which makes it seem to me further away.

Reply:
(sorry it's not normalized to [0;1] but to [0; width*100] I think. [0;1] is what I've chosen for knob)

Reply:
anyway, maybe you're right: changing then name `x_pos` to `x_val` could make the code more readable, as currently there's a clash between `x->x_pos` and `xpos = text_xpix(...)`.
should I?

Reply:
done with https://github.com/pure-data/pure-data/pull/933/commits/31fad0f4a1b430b3b82d5444f5b05174a95e35b5